docs/installation/pipx-based.rst

```diff
@@ -161,43 +188,45 @@ With ``pyenv`` installed and correctly configured, it is now easy to install any
 
     pyenv install -l
 
-You should see a very long list of versions to choose from. Let's install the latest version of the 3.9 family:
+You should see a very long list of versions to choose from. Let us install the latest version of the 3.9 family:
 
 .. code-block:: bash
 
-    pyenv install 3.9.13
+    pyenv install 3.9.19
```

Comment:
NB: On Linux you can write just `pyenv install 3.9`, but when I last tried this on Windows it didn't work, so I suppose we shouldn't advise users to do it